ci/docker/codecov/build.Dockerfile

```diff
@@ -80,7 +80,7 @@ RUN wget -q https://github.com/gperftools/gperftools/releases/download/gperftool
 
 # Install HPX
 ARG HPX_FORK=STEllAR-GROUP
-ARG HPX_VERSION=1.6.0
+ARG HPX_VERSION=master
```

Comment:
Please use a specific commit. With the master tag the build image building process breaks (due to caching)